src/Client/Spreadsheet/Controller/BuildingBlocks.fs

```diff
@@ -63,6 +63,30 @@ let addBuildingBlocks(newColumns: CompositeColumn []) (state: Spreadsheet.Model)
     table.AddColumns(newColumns,nextIndex)
     {state with ArcFile = state.ArcFile}
 
+let addDataAnnotation (data: {| fragmentSelectors: string []; fileName: string; fileType: string; targetColumn: DataAnnotator.TargetColumn |}) (state: Spreadsheet.Model) : Spreadsheet.Model =
+    let tryIfNone() =
+        match state.ActiveTable.TryGetInputColumn(), state.ActiveTable.TryGetOutputColumn() with
+        | Some _, None
+        | None, None        -> CompositeHeader.Output IOType.Data
+        | None, Some _      -> CompositeHeader.Input IOType.Data
+        | Some _, Some _    -> failwith "Both Input and Output columns exist and no target column was specified"
+    let newHeader =
+        match data.targetColumn with
+        | DataAnnotator.TargetColumn.Input -> CompositeHeader.Input IOType.Data
```

Comment:
allign the output and -> please, makes it easier to read

Reply:
Yeah another reason to setup fornax. A few more and i get to it! 😄